core/io/image.cpp

```diff
@@ -2631,9 +2631,10 @@ Vector<uint8_t> Image::save_jpg_to_buffer(float p_quality) const {
 }
 
 Error Image::save_exr(const String &p_path, bool p_grayscale) const {
-	if (save_exr_func == nullptr) {
-		return ERR_UNAVAILABLE;
-	}
+	ERR_FAIL_NULL_V_MSG(
+			save_exr_func,
+			ERR_UNAVAILABLE,
+			"The TinyEXR module isn't enabled. Recompile the Godot editor with the `module_tinyexr_enabled=yes` SCons option.");
```

Comment:
Note for reviewers: OpenEXR saving is only available in editor builds. Making it available in export templates has to be handled separately, e.g. using https://github.com/godotengine/godot/pull/73003.